cli/cli_test.go

```diff
@@ -353,3 +357,83 @@ func Example_max_results() {
 	// quit
 	// node drained and shutdown: ok
 }
+
+// TestFlagUsage is a basic test to make sure the fragile
+// help template does not break.
+func TestFlagUsage(t *testing.T) {
+	// Don't use cliTest as we want no other flags set.
+	osExit = func(int) {}
+	osStderr = os.Stdout
+
+	// Override os.Stdout with out own.
+	old := os.Stdout // keep backup of the real stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	if err := Run([]string{"help"}); err != nil {
+		fmt.Println(err)
+	}
+
+	outC := make(chan string)
+	// copy the output in a separate goroutine so printing can't block indefinitely
```

Comment:
uh. yes. quick search mentioned it may be as little as 512. I'll rework it to keep reading rather than plain copy.